.prettierrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    tabWidth: 4,
+    printWidth: 120,
```

Comment:
This needs to be increased or removed. It too often let into moving single values from into a separate line which makes them more unreadable than before (most for margins where each margin has a tool long variable name

Reply:
it defaults to 80, but can be set to `Infinity`

Reply:
done - is it really better now?

Reply:
Sometimes yes, sometimes not (lists in functions like linear-gradient)